dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -584,7 +604,10 @@ boolean isUnusedClassSymbolReference(ClassSymbolReference reference) {
           constantPoolIndexForClass(sourceJavaClass, targetClassName);
       Verify.verify(
           !targetConstantPoolIndices.isEmpty(),
-          "The target class symbol reference is not found in source class");
+          "The target class "
+              + targetClassName
```

Comment:
`Verify.verify` is varargs and uses a simple format.

```java
verify(
    !targetConstantPoolIndices.isEmpty(),
    "The target class %s is not found in source class %s",
    targetClassName,
    sourceJavaClass.getClassName());
```

The above is easier to read and can be more performant (which doesn't matter in this case) because string formatting doesn't happen unless verification fails.

Reply:
I disagree that format strings are easier to read than simple string concatenation. I like my words to come in the order I read them. Missing and out of order format string arguments are a common source of bugs. 

And to performance, I say [Knuth](http://wiki.c2.com/?PrematureOptimization)!

Reply:
I agree that string concatenation is just fine compared to adding `String.format()`.

For this case, I would use the varargs is for `verify()`; I like calling functions the way the implementation intends to.